pkg/api/k8s_versions.go

```diff
@@ -28,7 +28,7 @@ const (
 	antreaAgentImageReference                                = antreaControllerImageReference
 	antreaOVSImageReference                                  = antreaControllerImageReference
 	antreaInstallCNIImageReference                           = antreaControllerImageReference
-	azureNPMContainerImageReference                   string = "mcr.microsoft.com/containernetworking/azure-npm:v1.2.2_hotfix"
+	azureNPMContainerImageReference                   string = "mcr.microsoft.com/containernetworking/azure-npm:v1.4.16"
```

Comment:
The latest image we have is v1.4.22, lmk if you want me to make the change or you can continue.

Reply:
You are free to move this forward. Just one thing, I noticed that the current `cni` is v1.4.16:
https://github.com/Azure/aks-engine/blob/master/pkg/api/const.go#L322-L329

I am assuming it is better to keep `cni` and `nmp` in sync, but you probably know better how that works.

If you move to a different `nmp` version than v1.4.16 (using v1.4.22 as an example), then we should double check that
https://github.com/Azure/aks-engine/blob/master/parts/k8s/addons/azure-network-policy.yaml
is capturing changes in 
https://github.com/Azure/azure-container-networking/blob/v1.4.22/npm/azure-npm.yaml